src/platform/web/ui/css/layout.css

```
@@ -208,3 +208,13 @@ the layout viewport up without resizing it when the keyboard shows */
     min-height: 0;
     overflow-y: auto;
 }
+
+.LazyListParent {
+    height: 100vh;
```

Not sure why this doesn't make the lazy list too tall, but 100vh doesn't seem correct as there are buttons above it. This would be better:
```suggestion
    flex: 1;
```